vispy/scene/cameras/panzoom.py

```diff
@@ -133,7 +133,8 @@ def pan(self, *pan):
     @property
     def rect(self):
         """The rectangular border of the ViewBox visible area, expressed in
-        the coordinate system of the scene.
+        the coordinate system of the scene. See `~vispy.geometry.rect.Rect`
+        for different ways this can be specified.
```

Comment:
I think you need a `:class:` at the beginning of your Rect reference. Additionally, we should clean up this docstring to meet flake8 standards (current disabled on CI I think). What about...

```
"""Set the rectangular border of the ViewBox visible area.

This is expressed in the coordinate system of the scene. See :class:`~vispy.geometry.rect.Rect`
for different ways this can be specified.
```

Thoughts?